latest/schemas/axes.schema

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://ngff.openmicroscopy.org/latest/schemas/axes.schema",
+  "title": "NGFF Axes",
+  "description": "JSON from OME-NGFF .zattrs",
+  "type": "array",
+  "uniqueItems": true,
+  "items": {
+    "$ref": "#/$defs/axis"
+  },
+  "$defs": {
+    "axis": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the axis"
+        },
+        "longName": {
+          "type": "string",
+          "description": "Longer name or description of the axis."
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "array",
+            "space",
+            "time",
+            "channel",
+            "coordinate",
+            "displacement"
```

Comment:
How should schemas for this repository represent "MAY"?

Is this what's supposed to be in the `strict_*` schemas?

Reply:
The strict schemas are designed to give warnings that encourage more complete metadata (e.g. multiscales SHOULD have a "name"). If someone has used some axes `"type": "my_custom_type"` there is probably a good reason (and it maybe shouldn't fail a strict schema validation). *However* the spec does say  `It SHOULD be one of "array", "space", "time"...` and the strict schemas are used for SHOULD rules, so that is probably the right place to add that (rather than not include it at all).

Hopefully it's not too tricky to move it there?